source/macos/AdaptiveCards/AdaptiveCards/Views/ACRColumnView.swift

```diff
@@ -12,7 +12,9 @@ class ACRContentStackView: NSView, ACRContentHoldingViewProtocol {
     private (set) var stackViewTopConstraint: NSLayoutConstraint?
     private (set) var stackViewBottomConstraint: NSLayoutConstraint?
     let hostConfig: ACSHostConfig
+    public var backgroundView: NSView?
     var target: TargetHandler?
+    var style: ACSContainerStyle?
```

Comment:
This should be `private (set) var style: ACSContainerStyle` and this should be set inside `init` method.
This shouldn't be public.